code/FBX/FBXConverter.cpp

```diff
@@ -2508,28 +2767,24 @@ void FBXConverter::SetShadingPropertiesRaw(aiMaterial* out_mat, const PropertyTa
             // generate node animations
             std::vector<aiNodeAnim*> node_anims;
 
-            double min_time = 1e10;
-            double max_time = -1e10;
-
             int64_t start_time = st.LocalStart();
             int64_t stop_time = st.LocalStop();
             bool has_local_startstop = start_time != 0 || stop_time != 0;
-            if (!has_local_startstop) {
-                // no time range given, so accept every keyframe and use the actual min/max time
-                // the numbers are INT64_MIN/MAX, the 20000 is for safety because GenerateNodeAnimations uses an epsilon of 10000
-                start_time = -9223372036854775807ll + 20000;
-                stop_time = 9223372036854775807ll - 20000;
-            }
+
```

Comment:
This is where our frame truncation comes into play, we can declare start and stop times in blender/maya/max and they transparently come into our application without destroying any data.